rust/Cargo.lock

```diff
@@ -2893,7 +2893,7 @@ dependencies = [
  "httpdate",
  "itoa 1.0.10",
  "pin-project-lite",
- "socket2 0.5.6",
+ "socket2 0.4.10",
```

Comment:
??

Reply:
Odd! I'll see if I can undo that locally.